homeassistant/components/denonavr/strings.json

```diff
@@ -3,6 +3,7 @@
     "flow_title": "{name}",
     "step": {
       "user": {
+        "description": "By default this integration use a Telnet connection to your receiver to receive realtime updates. Note that only one telnet connection to your receiver can be established. This may interfere with your current setup. You can turn this option off after setting up the integration by clicking the Configure button.",
```

Comment:
```suggestion
        "description": "By default, this integration uses a Telnet connection to your receiver to receive real-time updates. Only one Telnet connection to your receiver can be established at a time. The Telnet connection can be disabled after setting up the integration.",
```

Reply:
The translator isn't likely to know how to translate `Configure button` so it matches what the button actually says

Reply:
Addressed in https://github.com/home-assistant/core/pull/85980/commits/08b7ab02c541dfe19ab1f26661fa60eff3244d45